.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       fail-fast: false
       matrix:
         platform: [ubuntu-18.04]
-        os_name: [linux_openresty, linux_tengine, linux_apisix_master_luarocks, linux_apisix_current_luarocks, linux_openresty_mtls]
+        os_name: [linux_openresty, linux_tengine, linux_apisix_master_luarocks, linux_apisix_current_luarocks, linux_openresty_mtls, linux_specialtest]
```

Comment:
what do you means of `specialtest`?

Reply:
I think this is a neutral term, which is convenient for adding other special cases in the future, such as long-term test, etcd link number monitoring test, etc.